### DISCUSSION TITLE
Docker environment variables

### DISCUSSION THREAD

**Discussion by itxtoledo** [dragonflydb/dragonfly#2089] [Q&A] [answered]
I don't find in documentation any available environment variables for docker. I need for example limit memory usage.

**Comment by romange** [comment]
Hi @itxtoledo, yes you are right that we do not have any examples nor explanation here: https://www.dragonflydb.io/docs/getting-started/docker
or here: https://www.dragonflydb.io/docs/getting-started/docker-compose   @Niennienzz  worth opening an issue about it and adding some examples on how to add command line arguments to docker or docker-compose.

but you can add regular arguments like this: `docker run --network=host --ulimit memlock=-1 docker.dragonflydb.io/dragonflydb/dragonfly --maxmemory=8GB`

**Reply by itxtoledo** [reply]
Would Dragonflydb not support reading environment variables in addition to passing parameters via the command line?

**Reply by chakaz** [reply, marked as answer]
We actually _do_ support reading environment variables!
Just set `DFLY_x` (where `x` is the exact name of the flag, case sensitive) and you should be good to go :)

**Reply by romange** [reply]
i.e. something like `docker run --network=host --ulimit memlock=-1 -e DFLY_maxmemory=8GB docker.dragonflydb.io/dragonflydb/dragonfly`

**Reply by itxtoledo** [reply]
It worked, thank you very much, but this could be in the documentation to save you time

**Reply by chakaz** [reply]
Great! Indeed, I'll add this to our README.
